pyvista/utilities/features.py

```diff
@@ -68,7 +68,7 @@ def voxelize(mesh, density=None, check_surface=True):
     x = np.arange(x_min, x_max, density_x)
     y = np.arange(y_min, y_max, density_y)
     z = np.arange(z_min, z_max, density_z)
-    x, y, z = np.meshgrid(x, y, z)
+    x, y, z = np.meshgrid(x, y, z, indexing='ij')
```

Comment:
Thank you for the change. I think we need an explanation (and also add a test if you can) of how the behavior has changed because of the change in the method.

Reply:
This change is made to make the ordering of the three dimensional ordering of the cell data and point data in the `StructuredGrid` as discussed in https://github.com/pyvista/pyvista/discussions/4360.

Is it suffice to explain here, or do I need to put explanation else where?

And I'm afraid adding test is beyond my capability, for now,

Reply:
The following may be helpful in adding explanations to the Docstring. Add it and then ask other maintainers to review it.
I will add the tests later on my end, could you add an explanation about your changes in docstring?
https://github.com/pyvista/pyvista/blob/5c40f80e1f64adbd215ce1589b534956191e19d7/pyvista/core/filters/poly_data.py#L2792-L2798

Reply:
Thanks for catching this, @natsuwater! This change is warranted, and the previous implementation must be corrected. I suspect this went unnoticed for as long as it did because this helper function isn't too widely used.

I wouldn't worry about testing this change -- we can handle that if it's really warranted

+1 for @tkoyama010's suggestion to add a note in the docstring. Something along the lines of "prior to version 0.39.0, this method improperly handled the structured coordinates"